--- gcl-2.6.13pre.orig/cmpnew/gcl_cmpmain.lsp	2022-11-09 01:43:51.000000000 +0100
+++ gcl-2.6.13pre/cmpnew/gcl_cmpmain.lsp	2022-11-09 12:05:56.000000000 +0100
@@ -483,7 +483,7 @@
 	  (t (setq dir ".")))
     (setq na  (namestring
 	       (make-pathname :name name :type (pathname-type(first args)))))
-   (format nil  "~a ~a -I~a ~a ~a -c ~a -o ~a ~a"
+   (format nil  "~a ~a -I~a ~a ~a -std=gnu89 -c ~a -o ~a ~a"
 	   (concatenate 'string
 			(if *prof-p* (remove-flag "-fomit-frame-pointer" *cc*) *cc*)
 			#+large-memory-model(if *large-memory-model-p* " -mcmodel=large " "")
