--- gcl-2.6.13pre.orig/h/386-macosx.defs	2022-06-16 12:11:29.000000000 +0200
+++ gcl-2.6.13pre/h/386-macosx.defs	2022-06-16 12:21:44.000000000 +0200
@@ -1,5 +1,9 @@
 # powerpc-macosx.defs
 
+# 20170416000 tpd link Axiom's code per Camm's instruction
+SYSTEM_OBJS = ${OBJ}/${SYS}/lib/cfuns-c.o ${OBJ}/${SYS}/lib/sockio-c.o
+OFLAG = -O
+
 # Disable Apple's custom C preprocessor which gets confused when
 # preprocessing some of the *.d files in the o/ subdirectory.
 CC = gcc $(CPPFLAGS)
