--- gcl-2.6.13pre.orig/o/unixfsys.c	2022-11-09 01:43:51.000000000 +0100
+++ gcl-2.6.13pre/o/unixfsys.c	2022-11-12 18:38:11.000000000 +0100
@@ -268,8 +268,12 @@
   memcpy(FN1,s->st.st_self,z1);
   FN1[z1]=0;
 #ifndef __MINGW32__
+#if _POSIX_C_SOURCE >= 200809L
   massert((l=readlinkat(d ? dirfd((DIR *)d) : AT_FDCWD,FN1,FN2,sizeof(FN2)))>=0 && l<sizeof(FN2));
 #else
+  massert((l=readlink(FN1,FN2,sizeof(FN2)))>=0 && l<sizeof(FN2));
+#endif
+#else
   l=0;
 #endif
   FN2[l]=0;
