--- gcl-2.6.13pre.orig/h/linux.defs	2022-06-16 12:11:29.000000000 +0200
+++ gcl-2.6.13pre/h/linux.defs	2022-06-16 12:22:57.000000000 +0200
@@ -8,6 +8,10 @@
 
 # Machine dependent makefile definitions for intel 386,486 running linux
 
+# 20120407000 tpd link Axiom's code per Camm's instruction
+SYSTEM_OBJS = ${OBJ}/${SYS}/lib/cfuns-c.o ${OBJ}/${SYS}/lib/sockio-c.o
+OFLAG = -O
+
 LBINDIR=/usr/local/bin
 
 #OFLAG	=  -g -Wall
