--- gcl-2.7.0pre.orig/unixport/makefile	2022-06-17 14:33:11.000000000 +0200
+++ gcl-2.7.0pre/unixport/makefile	2022-06-17 15:09:29.000000000 +0200
@@ -21,7 +21,7 @@
 LD_FLAGS:=$(LD_FLAGS) $(ODIR)/$(FIRST_FILE)
 endif
 LD_LIBS_PRE=$(addprefix -u ,$(PATCHED_SYMBOLS))
-LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp
+LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp ${OBJ}/${SYS}/lib/libspad.a
 ifneq ($(LAST_FILE),)
 LD_LIBS_POST:=$(LD_LIBS_POST) $(ODIR)/$(LAST_FILE)
 endif
