--- gcl-2.6.13pre.orig/unixport/makefile	2022-06-16 12:11:28.000000000 +0200
+++ gcl-2.6.13pre/unixport/makefile	2022-06-16 12:28:56.000000000 +0200
@@ -21,7 +21,7 @@
 LD_FLAGS:=$(LD_FLAGS) $(ODIR)/$(FIRST_FILE)
 endif
 LD_LIBS_PRE:=$(addprefix -u ,$(PATCHED_SYMBOLS))
-LD_LIBS_POST:=$(LIBS) $(LIBC) -lgclp
+LD_LIBS_POST:=$(LIBS) $(LIBC) -lgclp ${OBJ}/${SYS}/lib/libspad.a
 ifneq ($(LAST_FILE),)
 LD_LIBS_POST:=$(LD_LIBS_POST) $(ODIR)/$(LAST_FILE)
 endif
